.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,12 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.241.1/containers/docker-existing-dockerfile
+{
+	"name": "fedimint-dev",
+	"context": "..",
+	"image": "docker.io/rndhouse/fedimint-dev:latest",
```

Comment:
Long-term we probably don't want this pulling from your personal docker account. Probably fine for now. If this works well, then we might be able to have the post-merge CI tasks upload a docker image ...

Reply:
Definitely!

Reply:
I feel now that we are pretty close to getting docker CI running we might want to block on this to not forget about updating the dev container?